tests/host/Makefile

```diff
@@ -96,7 +96,7 @@ valgrind: $(OUTPUT_BINARY)
 	$(LCOV) --directory ../../cores/esp8266/ --zerocounters
 	$(VALGRIND) $(VALGRINDFLAGS) $(OUTPUT_BINARY)
 	$(LCOV) --directory $(CORE_PATH) --capture --output-file $(LCOV_DIRECTORY)/app.info
-	$(GENHTML) $(LCOV_DIRECTORY)/app.info -o $(LCOV_DIRECTORY)
+	-$(GENHTML) $(LCOV_DIRECTORY)/app.info -o $(LCOV_DIRECTORY)
```

Comment:
Is the hyphen here on purpose?

Reply:
Yes, it's on purpose because unfortunately LCOV throws an error due to the way we're using two different directories.  It runs and generates coverage properly, but complains that certain things can't be found (but it finds them elsewhere).  I wasn't able to figure out how to get rid of this w/o significant directory tree rework, which IMO's not worth the gried.